Sources/ComposableArchitecture/SwiftUI/TextState.swift

```diff
@@ -302,3 +302,51 @@ extension LocalizedStringKey: CustomDebugOutputConvertible {
     self.formatted().debugDescription
   }
 }
+
+extension TextState: CustomDebugOutputConvertible {
+  public var debugOutput: String {
+    func debugOutputHelp(_ textState: Self) -> String {
+      var output: String
+      switch textState.storage {
+      case let .concatenated(lhs, rhs):
+        output = debugOutputHelp(lhs) + debugOutputHelp(rhs)
+      case let .localized(key, tableName, bundle, comment):
+        output = key.formatted(tableName: tableName, bundle: bundle, comment: comment)
+      case let .verbatim(string):
+        output = string
+      }
+      for modifier in textState.modifiers {
+        switch modifier {
+        case .bold,
+             // TODO: Distinguish
+             .fontWeight(.some(.semibold)),
+             .fontWeight(.some(.bold)),
+             .fontWeight(.some(.heavy)),
+             .fontWeight(.some(.black)):
+          output = "**\(output)**"
```

Comment:
Currently doing some very basic Markdown-like formatting, but would probably be better to fully capture the text modifiers. Any ideas?

Reply:
only thing I can think of is XML tags 😬